src/main/java/com/software/modsen/rideservice/service/RideService.java

```diff
@@ -0,0 +1,22 @@
+package com.software.modsen.rideservice.service;
+
+import com.software.modsen.rideservice.dto.request.RideRequest;
+import com.software.modsen.rideservice.dto.response.RideResponse;
+import com.software.modsen.rideservice.dto.response.RideResponseSet;
+import com.software.modsen.rideservice.model.RideStatus;
+
+public interface RideService {
```

Comment:
Creating interfaces for only one implementation is a bad practice unless you have hierarchy or several implementations

Reply:
Removed the interface